components/ItemCard.tsx

```diff
@@ -11,6 +11,7 @@ type ItemCardPropType = {
   postcode?: string;
   postable?: boolean;
   itemId?: number;
+  reserved?: boolean;
```

Comment:
I really appreciate you going with the pre-set approach to the props but I can't for the life of me think why these were all set as optional props 😅 

I tried removing that and just navigating through the app for a little and nothing caught fire so can you make them all standard props instead please?

Reply:
I was confused by this as well, and they were actually causing an issue when debugging the code.
I believe it is actually better for them not to be optional, so very happy to change that.